index.html

```diff
@@ -133,17 +329,17 @@ <h3>
     </p>
     <p>
       Let <code>RTCRtpReceiver</code> objects have a
-      <dfn>[[\PlayoutDelayHint]]</dfn> internal slot initially
+      <dfn>[[\PlayoutDelay]]</dfn> internal slot initially
```

Comment:
Unrelated to this PR?

Reply:
Somehow github is showing diff against the wrong version. That change has been merged to head according to github.io

Reply:
Junior Oliveira